packages/hemera/index.d.ts

```diff
@@ -444,6 +444,18 @@ declare class Hemera<Request, Response> {
       instance: Hemera<Hemera.ServerRequest, Hemera.ServerResponse>
     ) => void
   ): Hemera<Request, Response>
+  on(
```

Comment:
The `this` context is of type `Hemera`

Reply:
Could you add it to all event handlers? https://www.typescriptlang.org/docs/handbook/functions.html#this-parameters

Reply:
Something like:

```js
  on(
    event: 'clientResponseError',
    handler: (
      this: Hemera<Hemera.ClientRequest, Hemera.ClientResponse>,
      error: Error
    ) => void
  ): void
```

Reply:
I will do it in a separate PR.

Reply:
done #216

Reply:
Hi @NAlexandrov my changes are already landed.